spec/controllers/guests_spec.cr

```diff
@@ -175,6 +179,62 @@ describe Guests do
     guest.extension_data.as_h.should eq({"test" => "data", "other" => "info"})
   end
 
+  with_server do
```

Comment:
Maybe add a `context "unique emails"` block, then you can do `before_all` and `after_all` blocks that do `WebMock.allow_net_connect = true` and `WebMock.reset`, respectively.